test_sets/SDL5_0/mobile_projection_2.txt

```diff
@@ -21,3 +21,6 @@
 ./test_scripts/MobileProjection/Phase2/021_two_apps_and_deactivation.lua
 ./test_scripts/MobileProjection/Phase2/022_two_apps_proj_media_and_deactivation_streaming.lua
 ./test_scripts/MobileProjection/Phase2/023_two_apps_navi_comm_and_deactivation_streaming.lua
+./test_scripts/MobileProjection/Defects/3139_01_Stop_streaming_gracefully_for_1st_app_if_2nd_is_activated_and_starts_streaming.lua
```

Comment:
I think that the adding of scripts for issue in test run for feature is not very idea. Sometimes core fix and scripts can be merged in different time so we can have the failed jobs in case scripts will be merged earlier. So i propose to add scripts to some separate test set. The same comment to smoke test run.

Reply:
@GetmanetsIrina I think the best that we can do for now is to remove links from existing test set(s).
And once scope for the next release (e.g. `6.1`) is finalized to create a specific test set for that release.